operations/use-log-cache.yml

```diff
@@ -1,8 +1,6 @@
-# safeguard: make sure there is no loggregator agent configured
 - type: remove
```

Comment:
Please revert the file renaming, this will cause breaking on customers side (e.g. SAP is already relying on this file to be named likethat)